compat/win32/fscache.c

```diff
@@ -171,13 +197,13 @@ static struct fsentry *fseentry_create_entry(struct fscache *cache, struct fsent
 		buf[off + fse->len] = '\0';
 	}
 
-	fse->st_mode = file_attr_to_st_mode(fdata->dwFileAttributes,
-			fdata->dwReserved0, buf);
+	fse->st_mode = file_attr_to_st_mode(fdata->FileAttributes,
+			fdata->EaSize, buf);
```

Comment:
This is a bit unintuitive, and I think you talked to @ChandKV about this so you know that `EaSize` now plays the role of what `dwReserved0` did. Is there any public documentation to which we could link? And: could you maybe add a short paragraph to the commit message talking about the conditions when `EaSize` do *not* contain any size (but a reparse tag instead)?

Reply:
I'll add links to the docs in the commit message and the PR description.